data-prepper-plugin-framework/src/main/java/org/opensearch/dataprepper/plugin/ExtensionsConfiguration.java

```diff
@@ -0,0 +1,7 @@
+package org.opensearch.dataprepper.plugin;
```

Comment:
We should probably change this package to be something else. But, maybe in another PR.